packages/material-ui/src/Paper/Paper.js

```diff
@@ -12,6 +25,13 @@ export const styles = (theme) => {
     };
   });
 
+  const overlays = {};
+  theme.shadows.forEach((_, index) => {
+    overlays[`overlay${index}`] = {
+      backgroundColor: fade(theme.palette.background.paper, calculateAlpha(index)),
```

Comment:
According to the dark theme page in the Material website the surface color changes by using a white (`#fff`) surface color with transparency. It seems to me that what you are doing here is adding transparency to the paper whose color is `#121212`- causing the paper to be transparent, unless I'm missing something of course.